docs/development/Building in Ubuntu.md

```diff
@@ -8,7 +8,9 @@ This document is tested and based on the latest Ubuntu 20.04 LTS release.
 ```
 sudo apt update
 sudo apt upgrade
-sudo apt install build-essential
+sudo apt install build-essential python clang-8 libblocksruntime-dev
```

Comment:
The python requirement is a bit ridiculous really - all we need this for is to enable parasitic installation of the ARM toolchain. If we wanted to do this along the same path we should be downloading and unpacking python inside the repository as well instead, alongside with clang and clang++.
I propose we hold off on merging this while we investigate if the parasitic installation can be changed to work without python. But thanks for investigating and figuring out that this currently uses tooling that is not installed by default.

Reply:
Yeah, you are right, we do use python in the Makefile - `dfuse-pack.py` is used to prevent the `<target>_flash` target from overwriting the Unified Target configuration on targets where this is possible.
Until now we are just not actually using the `PYTHON` environment variable that generates the warning when python is not found. #10266 fixes this.

Reply:
See my notes in PR10266. I can make a PR for those changes if you like.